lib/write.js

```diff
@@ -17,15 +17,16 @@ var Write = Juttle.proc.sink.extend({
     initialize: function(options, params) {
         this.name = 'write-influx';
 
-        var allowed_options = ['raw', 'db', 'intFields', 'valFields', 'measurementField', 'measurement'];
+        var allowed_options = ['raw', 'db', 'intFields', 'valFields', 'nameField', 'name'];
```

Comment:
I don't think it makes sense for `write` to support a `-name` option.

If a user wants to set all points to the same name, they should do `put name='something' | write influx ...` as opposed to `write influx -name 'something'`.

To that end, it should be a runtime error if points arrive at the write with nothing in the `nameField` and we should simply skip the points.